src/pkg/data/consumers/governor/exporter.go

```diff
@@ -75,13 +75,12 @@ func (g GovernorExporter) getGovernorAPIPayload() openapi.KubernetesTelemetryReq
 			kubernetesWorkloads.Name = workloadAssessment.Workload.Name
 			kubernetesWorkloads.Kind = workloadAssessment.Workload.Kind
 			kubernetesWorkloads.Namespace = nsa.Namespace.Name
-			kubernetesWorkloads.Replicas = workloadAssessment.Workload.Replicas
 
 			for _, pod := range workloadAssessment.Workload.Pods {
 				containerData := openapi.NewContainerWithDefaults()
 				for _, container := range pod.Containers {
 					containerData.Name = container.Name
-					containerData.Id = container.ID
```

Comment:
containerId is temporarily set as optional filed in the governor backend. Please manually change the go-client to treat it as a mandatory field.

Reply:
Auto generated file should not be touched like this, instead let me know once it is mandatory in backend and i will update the go-client using auto generation.

Reply:
Let's keep it as it is. Additionally keep in mind this is a change in 0.2.1 branch, which was a temporary one until the new architecture was merged in the main one (which already happened).